src/action/expense.js

```diff
@@ -0,0 +1,20 @@
+import uuid from 'uuid/v4';
+
+export const createAction = ({ name, categoryID }) => ({
```

Comment:
code is breaking here, if you change `categoryID` to `categoryId` (lower case 'd') it works.

This is because your expense reducer is expecting a property of `...Id` not `...ID`